dart/lib/src/sentry_envelope_item.dart

```diff
@@ -12,21 +13,12 @@ class SentryEnvelopeItem {
 
   /// Creates an [SentryEnvelopeItem] which sends [SentryTransaction].
   factory SentryEnvelopeItem.fromTransaction(SentryTransaction transaction) {
-    final cachedItem = _CachedItem(() async {
-      final jsonEncoded = jsonEncode(
-        transaction.toJson(),
-        toEncodable: jsonSerializationFallback,
-      );
-      return utf8.encode(jsonEncoded);
-    });
-
-    final getLength = () async {
-      return (await cachedItem.getData()).length;
-    };
+    final cachedItem =
```

Comment:
If we are using always CachedItem, we can close https://github.com/getsentry/sentry-dart/issues/812 ?